pep/app/services/current-user.ts

```diff
@@ -255,4 +268,87 @@ export default class CurrentUserService extends Service {
         const prefs = this.preferences;
         return prefs?.[key] ?? [];
     }
+
+    /**
+     * Set new font size on html element
+     *
+     * @param {FontSizes} newSize
+     * @memberof CurrentUserService
+     */
+    setFontSize(newSize: FontSizes) {
+        const document = this.document;
+        let target = document.documentElement;
+        const size = AvailableFontSizes.find((item) => item.id === newSize) ?? FONT_SIZE_DEFAULT;
+        AvailableFontSizes.map((item) => item.class).forEach((item) => {
+            removeClass(target, item);
+        });
+
+        addClass(target, size.class);
```

Comment:
we should be able to remove the need for this imperative manipulation of the document classes by utilizing ember-body-class2, which is already a dep of the project and is fastboot-safe as well.

I believe it allows dynamic updating of the class via getters, if so this could be reduced to just adding the following on the `application` route:
```ts
get classNames() {
    //get the current fontsize from the currentUser service here...
    return currentFontSize.class
}
```

Note though, that this approach may or may not be viable if we need to set the styles on the `<html>` element instead like i mention in my other comment regarding the font-size styles. But this declarative approach _would_ be much nicer.

Reply:
Correct, I originally tried that route but it needed to be on the HTML element, not the body.

Reply:
👍  gotcha yeah i missed the fact that the document element referred to the HTML element, not the body.

ideally, it would be nice if we could override the ember-body-class2 code to allow the html element to be the target instead of `<body>` here: https://github.com/nathanhammond/ember-body-class/blob/master/addon/utils/body-class.js#L68  so that we can keep our app code cleaner and declarative, but i guess we can punt on that for now.

Reply:
Yes I originally tried to use that, but when I did I was getting build errors that the package didn't exist. I played with it for a bit before giving up. Figured we can iterate later if need bo